worlds/sm64ex/Options.py

```diff
@@ -127,6 +127,30 @@ class MoveRandomizerActions(OptionSet):
     valid_keys = [action for action in action_item_table if action != 'Double Jump']
     default = valid_keys
 
+sm64_options_groups = [
+    OptionGroup("Logic Options", [
+        BuddyChecks,
+        ExclamationBoxes,
+        ProgressiveKeys,
+        EnableCoinStars,
+        EnableMoveRandomizer,
+        MoveRandomizerActions,
+        StrictCapRequirements,
+        StrictCannonRequirements,
+        StrictMoveRequirements,
+    ]),
+    OptionGroup("Goal Options", [
```

Comment:
The only actual goal option is `CompletionType` (and maybe `StarsToFinish`), the rest arguably are not.
I'd say you could call this `Star Options` / `Star Costs` instead, and remove `CompletionType` from that category

Reply:
Added `Star Options` group and separated from Goal Option(s).